Solutions/DigitalGuardianDLP/Analytic Rules/DigitalGuardianFileSentToExternalDomain.yaml

```diff
@@ -0,0 +1,34 @@
+id: a19885c8-1e44-47e3-81df-d1d109f5c92d
+name: Digital Guardian - Exfiltration to external domain
+description: |
+  'Detects exfiltration to external domain.'
+severity: Medium
+requiredDataConnectors:
+  - connectorId: DigitalGuardianDLP
+    dataTypes:
+      - DigitalGuardianDLPEvent
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Exfiltration
+relevantTechniques:
+  - T1048
+query: |
+  let corp_domain = dynamic(['example.com']);

```

Comment:
Please add a comment to let users know to add their own domains.

Reply:
added comment